hdf5-src/Cargo.toml

```diff
@@ -30,6 +30,7 @@ hl = []
 zlib = ["libz-sys"]
 deprecated = []
 threadsafe = []
+default = ["deprecated"]
```

Comment:
Wondering why add this?

Reply:
In `src/hl/dataspace.rs` we are using `H5Sencode1` which is not included when building `hdf5-1.13` from source

Reply:
Hm, shouldn't we rather have a `cfg_if` there so that `H5Sencode1` is only used pre-1.13?